test/fixtures/expect.go

```diff
@@ -41,6 +41,26 @@ type Expect struct {
 	kubeClient     kubernetes.Interface
 }
 
+func (t *Expect) RedisContains(sinkName string, targetRegex string, opts ...RedisCheckOption) *Expect {
```

Comment:
Maybe use `RedisSinkContains` in the following up PR?

Reply:
Renamed it to SinkContains in [PR](https://github.com/numaproj/numaflow/pull/436/files#diff-69c8655fa539b97eeb80a600bc0d6bd0e0df96855eee7b2be8ae802dda68f094R44) as per discussion: https://github.com/numaproj/numaflow/pull/434#discussion_r1054647145